extension/src/shared/ui-components/FeedBackModal.tsx

```diff
@@ -0,0 +1,210 @@
+import { FormControl } from "@mui/base/FormControl";
+import { Input, inputClasses } from "@mui/base/Input";
+import { TextareaAutosize } from "@mui/base/TextareaAutosize";
+import CancelIcon from "@mui/icons-material/Cancel";
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControlLabel,
+  IconButton,
+  Link,
+  Modal,
+  Typography,
+  styled,
+} from "@mui/material";
+import { red } from "@mui/material/colors";
+import { useCallback, useMemo, useState } from "react";
+
+const style = {
```

Comment:
Also could you support mobile as well? I'm sorry I forgot to mention about this...
You can set breakpoint like below.
```
const StyledBox = styled(Box)({ theme }) => ({
  // ... other styles
  [theme.breakpoints.down("mobile")]: {
    width: `calc(100vw - ${theme.spacing(2)})`,
   // Also it should be scrollable if the height is greater then the window height.
  },
})
```